lib/web_ui/lib/src/engine/web_experiments.dart

```diff
@@ -58,7 +58,7 @@ class WebExperiments {
   }
 
   /// Used to enable/disable experimental flags in the web engine.
-  void updateExperiment(String name, bool enabled) {
+  void updateExperiment(String name, bool? enabled) {
```

Comment:
One of our tests is specifically passing `null` here. Also the setters for `useCanvasText` and `useCanvasRichText` accept null. So I'm assuming the expectation is that this must support `null`.